SPECS/libnbd/CVE-2024-7383-1.patch

```diff
@@ -0,0 +1,58 @@
+From c6cc19319f39c09c4ff74b47101eb217e75d3b43 Mon Sep 17 00:00:00 2001
```

Comment:
How about we combine all the patch hunks in a single patch:
```bash
cat CVE-2024-7383-* > CVE-2024-7383.patch
```

Reply:
sure ... wasn't sure how to combine patches, but will try your suggestion.

Reply:
combined into one.